kpi/deployment_backends/mixin.py

```diff
@@ -239,25 +240,29 @@ def validation_status(self, kpi_request, pk, *args, **kwargs):
         http_method_params = {}
         if kpi_request.method == "PATCH":
             http_method_params = {"json": kpi_request.data}
-        else:
+        elif kpi_request.method == "GET":
```

Comment:
This changes the semantics of the code. I think you should add a `else` block, and if it should never be run, then either throw an error, or log something.

Reply:
@noliveleger this is your change - what do you think? :)

Reply:
IMHO, there is no needs to add an `else` because method can be only `PATCH`, `GET` or `DELETE`.
`if elseif` is only there to override `http_method_params`.

@Eric-Dilcher would you prefer something like this?
```
if kpi_request.method == "PATCH":
    http_method_params = {"json": kpi_request.data}	            http_method_params = {"json": kpi_request.data}
elif kpi_request.method == "GET":
    http_method_params = {"params": kpi_request.GET}
else:
    http_method_params = {}
```

Reply:
This should be okay as-is because of the Django REST Framework `@detail_route` decorator. The `else` case is a deliberate no-op.